app/controllers/catalog_controller.rb

```diff
@@ -1,6 +1,8 @@
 class CatalogController < SubsitesController
 
   before_action :refresh_catalog_browse_lists_cache, only: [:home, :browse]
+  
+  self.solr_search_params_logic += [:hide_concepts_when_query_blank_filter]
```

Comment:
Thanks, @cgalarza!  The only reason I didn't add this to search_params_logic_behavior.rb is because the filter only applies to CatalogController.  search_params_logic_behavior.rb is included in all of our various subsites via the misleadingly named catalog_includes.rb, which is included by subsites_controller.rb.

We need to do a big restructuring of DLC at some point.  Unfortunately, "catalog" means "generic blacklight catalog functionality" in some cases and "main DLC catalog subsite" in others.